inference-engine/thirdparty/clDNN/src/CMakeLists.txt

```diff
@@ -128,7 +128,15 @@ target_link_libraries("${CLDNN_BUILD__PROJ}" PRIVATE
 )
 
 if(ENABLE_ONEDNN_FOR_GPU)
-  target_link_libraries("${CLDNN_BUILD__PROJ}" PUBLIC onednn_gpu)
+  get_property(isMultiConfig GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
+  if (isMultiConfig)
+    add_dependencies(${CLDNN_BUILD__PROJ} onednn_gpu_build)
+	foreach(dir IN LISTS ONEDNN_INCLUDE_DIRS)
+	  target_include_directories("${CLDNN_BUILD__PROJ}" PUBLIC $<BUILD_INTERFACE:${dir}>)
+	endforeach()
+	target_compile_definitions("${CLDNN_BUILD__PROJ}" PUBLIC ENABLE_ONEDNN_FOR_GPU)
+  endif()
+  target_link_libraries("${CLDNN_BUILD__PROJ}" PUBLIC ${ONEDNN_LIBRARY})
```

Comment:
why do we need changes here?

Reply:
Generator expressions don't work correctly together with set_target_properties().
Thus,  properties and dependencies onednn_gpu_build have to be re-assigned by hand.

Reply:
how they are working in current master?

Reply:
On master project properties and dependencies are passed through single global "onednn_gpu" library target.
The need to use generator expressions disables such a convenient option.